### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer without…

### PR DESCRIPTION
**Pull request by OVikhrov**
… seconds parameter
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [MOBILE_API] New OnSeekMediaClockTimer notification
+-- [HMI_API] New OnSeekMediaClockTimer notification
+-- 
+--
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_Without_seconds_Of_seekTime()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {hours = 30, minutes = 30}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua [hunk 3]
```diff
@@ -0,0 +1,96 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua]: @OVikhrov, please update Requirement summary with related functional requirement also

Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Absent.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**